src/index.js

```diff
@@ -60,6 +60,23 @@ class PaperWallet {
     return { claimData: claimData, qr: qr, version: ver }
   }
 
+  /**
+   * creates the wallet
```

Comment:
So maybe let's refactor this a little to make it more generic. The reason I made this function is to be able to create a QR code with raw-claim data, instead of using the export function. In order not to break the rendering, I wanted to be able to re-use the claimdata from an issue call. So maybe let's call this "rawIssue" or something?